pom.xml

```diff
@@ -101,7 +101,7 @@
 		<dependency>
 			<groupId>com.sk89q</groupId>
 			<artifactId>worldedit</artifactId>
-			<version>5.5.8</version>
+			<version>5.6.3</version>
```

Comment:
The current version is the embed version of the current world guard, but it has the conflicts before the pom file update commit. I fixed these conflicts by exlucuding all conflicting dependencies and adding it to pom file manually, at the end.

But i'm missed this, the world edit version is slightly old. The 5.6.3's changelog says "Added support for 1.7.10", the LifeSkript project is started on my server and i published for other users using older versions because of their old servers, and can not update because some datas and plugins will corrupt. So the project's main goal is compatibility with everything. And the main supported versions are 1.7-1.8 versions of Minecraft and JRE.

The WorldEdit is a runtime dependency and it does not affect runtime, so we can update safe. But IF anything we used in the project IF changed in the API; this breaks.

I'm merging this now, if i found a compatibility problem, maybe a fix (with supporting old and new world edit versions via reflection etc) but i think no breaking changes done because this a minor update.